test/test-annotation.js

```diff
@@ -303,6 +323,15 @@ describe('Annotations', function () {
       return util.checkDatabase((resolve) => {
         resolve(Annotation.findById(StupidAnnotation.id).should.eventually.be.null);
       });
+
```

Comment:
keeping this here because I was using it to test out pagination and mongodb. But will remove once all pagination is implemented.